app/src/androidTest/java/com/android/unio/Utils.kt

```diff
@@ -8,17 +8,22 @@ import androidx.compose.ui.test.onNodeWithTag
 import androidx.compose.ui.test.performClick
 import androidx.compose.ui.test.performScrollTo
 import androidx.compose.ui.test.performTextInput
+import com.android.unio.model.authentication.unregisterAllAuthStateListeners
 import com.android.unio.model.strings.test_tags.SocialsOverlayTestTags
+import com.google.firebase.Firebase
+import com.google.firebase.auth.auth
+import io.mockk.clearAllMocks
+import io.mockk.unmockkAll
+import org.junit.After
 
 /*
  * Scrolls to a component if it's not displayed and asserts if it is displayed
  */
-
-fun assertDisplayComponentInScroll(compose: SemanticsNodeInteraction) {
-  if (compose.isNotDisplayed()) {
-    compose.performScrollTo()
+fun SemanticsNodeInteraction.assertDisplayComponentInScroll() {
```

Comment:
Is there a reason why you made this change ?

Reply:
Yes. Although the behavior is still the same, this method signature avoids passing a SemanticsNodeInteraction as parameter to another function because that is generally considered bad practice, as the method thus would share state between tests.